internal/docs/generated/fndocs/docs.go

```diff
@@ -53,6 +56,34 @@ var ExportExamples = `
   kpt fn export DIR/ --fn-path FUNCTIONS_DIR/ --workflow cloud-build
 `
 
+var RenderShort = `Render a package.`
+var RenderLong = `
+  kpt fn render [PKG_PATH] [flags]
+
```

Comment:
One thing I noticed that we don't include the short description in the commandline reference. Not sure if this is bug or intentional. Having it it as part of CLI reference will be useful IMO.

/cc @mortent @etefera